packages/docs/src/examples/styles/styles.module.css

```diff
@@ -0,0 +1,19 @@
+#Account {
+  margin-left: 5px;
+  font-weight: 450;
+}
+
+#Balance {
+  font-size: 16px;
+  color: grey;
+  display: inline-flexbox;
+}
+
+#Bold {
```

Comment:
You can have more than 1 bold item on a page, so this should be a class not an ID - same with others.